css/civicrm.css

```diff
@@ -270,7 +270,7 @@ input.crm-form-entityref {
   border-top: 1px solid #696969;
 }
 
-/* Size the verticle heights in crm-containers by class. */
+/* Size the vertical heights in crm-containers by class. */
```

Comment:
I kinda liked verticle - I feel like little verticles could be hanging down from my ceiling